### PR TITLE
Ssp585 hist ghg v1.0

### PR DESCRIPTION
**Pull request by chengzhuzhang**
Add meta data definition files for: 3 ensembles of hist-GHG, and 5 ensembles of ssp585 E3SM v1 runs.

### CODE REVIEW

File: E3SM-1-0/hist-GHG_r1i1p1f1.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  "parent_variant_label": "r1i1p1f1",
+
+  "parent_time_units": "days since 0001-01-01",
+
+  "branch_method": "standard",
+
+  "branch_time_in_child": 0.0,
+
+  "branch_time_in_parent": 36500.0,
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r1i1p1f1.json:40]: Please check.

File: E3SM-1-0/hist-GHG_r2i1p1f1.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  "parent_variant_label": "r1i1p1f1",
+
+  "parent_time_units": "days since 0001-01-01",
+
+  "branch_method": "standard",
+
+  "branch_time_in_child": 0.0,
+
+  "branch_time_in_parent": 54750.0,
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r2i1p1f1.json:40]: Please check

File: E3SM-1-0/hist-GHG_r3i1p1f1.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  "parent_variant_label": "r1i1p1f1",
+
+  "parent_time_units": "days since 0001-01-01",
+
+  "branch_method": "standard",
+
+  "branch_time_in_child": 0.0,
+
+  "branch_time_in_parent": 73000.0,
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r3i1p1f1.json:40]: Please check

**Review by chengzhuzhang — COMMENTED** [review]
@tangq Hi Qi, i drafted the metadata files for submitting 7 v1 simulations. Would you please gloss over? For the hist-GHG runs, i suppose the branch time from piControl is identical for the first three historical runs? Please confirm, thank you!

**Comment by chengzhuzhang** [comment]
@TonyB9000 Just FYI, once this PR is merged, we will need update the local copy.

**Comment by TonyB9000** [comment]
OK - the URL reads "E3SM-Project/CMIP6-Metadata/pull/5".  I cannot tell if this is "warehouse" or "e3sm_to_cmip" or some other repo.  Is "CMIP6-Metadata" a repo?  I've never cloned it, so how would I "update the local copy"?

**Comment by chengzhuzhang** [comment]
It's a data repo that both warehouse and e3sm_to_cmip depend on. I think you have already cloned it to staging directory on acme1.

**Comment by TonyB9000** [comment]
Ah, ok.  I know which one you mean.  Thanks!  Will this affect my current postprocess testing?

**Comment by chengzhuzhang** [comment]
nope. you can go ahead with current testing.

**Comment by chengzhuzhang** [comment]
@tangq Pinging for a review...Thanks.

**Comment by tangq** [comment]
@chengzhuzhang , I missed this review request a few days ago.

I saw you put a few checking places above. Do you mean the "branch_time_in_parent" field?

The simulations are documented on these two confluence pages. All the details should be there.
https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/958332992/DAMIP+hist-GHG+simulations
https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/1027146101/DECKv1+future+projection+simulations

**Comment by chengzhuzhang** [comment]
Hi Qi, thanks for pointing to the pages. 
yes, I'd like to make sure if the "branch_time_in_parent" for hist-GHG are identical as that for historical runs from pi-Control.

**Review by tangq — APPROVED** [review]
Looks good by visual check.

File: E3SM-1-0/hist-GHG_r1i1p1f1.json [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+  "outpath": "CMIP6",
+
+  "#note_optional": " **** The following descriptors are optional and may be set to an empty string ",
+
+  "project PI": "Dave Bader (bader2@llnl.gov)",
+
+  "data contact": "E3SM-DATA-SUPPORT@LLNL.GOV",
+
+  "history": "output from 20190404.DECKv1b_H1_hist-GHG.ne30_oEC.edison",
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r1i1p1f1.json:70]: here indicates original casename

File: E3SM-1-0/hist-GHG_r2i1p1f1.json [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+  "outpath": "CMIP6",
+
+  "#note_optional": " **** The following descriptors are optional and may be set to an empty string ",
+
+  "project PI": "Dave Bader (bader2@llnl.gov)",
+
+  "data contact": "E3SM-DATA-SUPPORT@LLNL.GOV",
+
+  "history": "output from 20190404.DECKv1b_H2_hist-GHG.ne30_oEC.edison",
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r2i1p1f1.json:70]: For original case name

File: E3SM-1-0/hist-GHG_r3i1p1f1.json [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+  "outpath": "CMIP6",
+
+  "#note_optional": " **** The following descriptors are optional and may be set to an empty string ",
+
+  "project PI": "Dave Bader (bader2@llnl.gov)",
+
+  "data contact": "E3SM-DATA-SUPPORT@LLNL.GOV",
+
+  "history": "output from 20190404.DECKv1b_H3_hist-GHG.ne30_oEC.edison",
```
Comment (chengzhuzhang) [inline E3SM-1-0/hist-GHG_r3i1p1f1.json:70]: For original casename

File: E3SM-1-0/hist-GHG_r3i1p1f1.json (hunk 3 above)
Comment (chengzhuzhang) [reply E3SM-1-0/hist-GHG_r3i1p1f1.json:40]: yes

File: E3SM-1-0/hist-GHG_r2i1p1f1.json (hunk 2 above)
Comment (chengzhuzhang) [reply E3SM-1-0/hist-GHG_r2i1p1f1.json:40]: yes

File: E3SM-1-0/hist-GHG_r1i1p1f1.json (hunk 1 above)
Comment (chengzhuzhang) [reply E3SM-1-0/hist-GHG_r1i1p1f1.json:40]: Yes, the file name hist-GHG_r1i1p1f1, and the `history` attribute gives original case name, now I also highlighted.

**Comment by chengzhuzhang** [comment]
Thank you for reviewing @tangq ! Merging.

**Comment by chengzhuzhang** [comment]
@TonyB9000 Hey Tony, Please pull the master for the updates. Thanks.

**Comment by TonyB9000** [comment]
OK, done!

**Pull request merged**